Makefile

```diff
@@ -51,9 +52,15 @@ BUILD_OPTS_DEPLOY?="-ldflags=$(BUILDINFO) -w -s"
 
 check: lint test ## Run linters and tests
 
-build: host-apps bin ## Build apps and binaries. `go build` with ${OPTS}
+move-built-frontend:
+	rm -rf ${MANAGER_UI_BUILT_DIR}
+	mkdir ${MANAGER_UI_BUILT_DIR}
+	cp -r ${MANAGER_UI_DIR}/dist/. ${MANAGER_UI_BUILT_DIR}
+	rm -rf ${MANAGER_UI_DIR}/dist/
```

Comment:
I’m not sure if this is needed, as that folder is already removed from the repo. It indeed prevents having duplicated files, but also has the potential of forcing to rebuild the UI if the folder is needed again for any reason.

Reply:
removed the last line